src/h5web/visualizations/heatmap/hooks.ts

```diff
@@ -14,10 +14,10 @@ import { TextureWorker } from './worker';
 import { useVisProps } from '../../dataset-visualizer/VisProvider';
 
 export function useData(): number[][] {
```

Comment:
I wonder if this hook is still needed. If the code is correct in the provider, then the check and throw are superfluous. If `rawDims` doesn't have a length of 2, the vis will break anyway.